features/promotion/managing_promotions/adding_promotion.feature

```diff
@@ -0,0 +1,18 @@
+@managing_promotions
+Feature: Adding a new promotion
+    In order to give possibility to pay less for some goods
+    As an Administrator
+    I want to add a new promotion to the registry
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new promotion
```

Comment:
This scenario does not make sense, because we should probably have a validation for "at least 1 action is required". Promotion without action does not work very well.